staging/src/k8s.io/kubectl/pkg/cmd/create/create_namespace.go

```diff
@@ -60,35 +99,122 @@ func NewCmdCreateNamespace(f cmdutil.Factory, ioStreams genericclioptions.IOStre
 		},
 	}
 
-	options.CreateSubcommandOptions.PrintFlags.AddFlags(cmd)
+	options.PrintFlags.AddFlags(cmd)
 
 	cmdutil.AddApplyAnnotationFlags(cmd)
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, generateversioned.NamespaceV1GeneratorName)
```

Comment:
Please:
* Remove this line (AddGeneratorFlag)
* Add cmdutil.AddDryRunFlag(cmd)